history/characters/62000_dragon.txt

```diff
@@ -592,7 +592,6 @@ darkblaze={
 
 azuregos={
 	name=Azuregos
-	dynasty=azurewing
```

Comment:
Low Born ?

Reply:
[He doesn't really have a last name](https://warcraft.wiki.gg/wiki/Azuregos)

Reply:
No dragons has a last name

Reply:
Well some do, be it the [name of their brood](https://warcraft.wiki.gg/wiki/Senegos), dragonflight or the [last name that their visage has](https://warcraft.wiki.gg/wiki/Darkblaze).
Azuregos doesn't have any of those from what I saw.

Reply:
Well, ok